resources/dicts/thoughts/alive/general.json

```diff
@@ -771,78 +831,50 @@
             "Suffered a bellyache after being dared to swallow a beetle",
             "Is twitching {PRONOUN/m_c/poss} tail in excitement about something",
             "Wishes there were more opportunities to show {PRONOUN/m_c/poss} skills",
-            "Defended a Clanmate in an argument"
-        ],
-        "main_trait_constraint": [
-            "daring"
-        ],
-        "main_status_constraint": [
-            "kitten",
-            "apprentice",
-            "medicine cat apprentice",
-            "mediator apprentice",
-            "warrior",
-            "medicine cat",
-            "mediator",
-            "deputy",
-            "leader",
-            "elder"
-            ]
-    },
-    {
-        "id": "daring_not_kit",
-        "thoughts": [
+            "Defended a Clanmate in an argument",
             "Is letting a wound go untreated in hopes it gives {PRONOUN/m_c/object} a scar",
```

Comment:
looks like these two sets of daring thoughts were separated by one being appropriate for kits and the other not.  You should separate them out again accordingly, please

Reply:
kits can't have any of these thoughts anyways since daring (and lonesome) aren't in the kit traits dict of trait-ranges.json anymore?

Reply:
Ahh, you're right! I totally forgor that